pymove/core/grid.py

```diff
@@ -14,13 +14,16 @@
     LONGITUDE,
     POLYGON,
     TID,
-    TRAJ_ID)
+    TRAJ_ID,
+)
 from pymove.utils.conversions import lat_meters
 from pymove.utils.mem import begin_operation, end_operation
 
 
-class Grid():
-    def __init__(self, data, cell_size, meters_by_degree = lat_meters(-3.8162973555)):
+class Grid:
+    def __init__(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5421338)